src/common/config/globalOptions.js

```diff
@@ -11,3 +56,88 @@ const defaultConfig = (ctx) => {
 };
 
 export default defaultConfig;
+
+/**
+ * The function `handleLegendHover` updates the background and border colors of a chart legend item
+ * when it is hovered over.
+ * @param e - The `e` parameter is an event object that represents the event that triggered the
+ * function. It can be used to access information about the event, such as the target element or the
+ * event type.
+ * @param item - The `item` parameter represents the legend item that was hovered over. It contains
+ * information about the dataset index and the index of the hovered item within that dataset.
+ * @param legend - The `legend` parameter is the legend object of a chart. It contains information
+ * about the chart's legend, such as the labels and colors of the legend items.
+ */
+const handleLegendHover = (e, item, legend) => {
+  const DatasetIndex = item.datasetIndex || 0;
+  const Datasets = legend.chart.data.datasets;
+  const Dataset = Datasets[DatasetIndex];
+
+  if (Array.isArray(Dataset.backgroundColor)) {
+    Dataset.backgroundColor.forEach((color, index, colors) => {
+      colors[index] =
+        index === item.index || color.length === 9 ? color : color + '4D';
```

Comment:
I think it'd better to assign number 9 and such things a descriptive constant name so they won't be magic numbers.